client/src/components/Header/Header.js

```diff
@@ -1,63 +1,22 @@
 import React from "react";
-import PropTypes from "prop-types";
+import styled from "styled-components";
+import LogoHeader from "../../../src/assets/header_logo_JRK.svg";
 
-import Button from "../Button";
-import "./header.css";
+const AppHeader = styled.header`
+  display: flex;
+  justify-content: space-between;
+  img {
+    height: 55px;
+    padding: 0 2rem;
+  }
+`;
 
-const Header = ({ user, onLogin, onLogout, onCreateAccount }) => (
-  <header>
+const Header = () => (
+  <AppHeader>
```

Comment:
It sounds wrong that the `Header` contains a `AppHeader`. I would swap the names and call this file `AppHeader`.

Reply:
Done. Thanks